packages/hebao_v2/package.json

```diff
@@ -12,11 +12,11 @@
   },
   "devDependencies": {
     "@nomiclabs/hardhat-ethers": "^2.0.1",
-    "@nomiclabs/hardhat-waffle": "^2.0.1",
+    "@nomiclabs/hardhat-waffle": "^1.0.0",
```

Comment:
What is the purpose of lowering the minimum version for some of these packages?

Reply:
This PR is not ready. I'm sure how the above happened ：）